### PR TITLE
[FIX] don't require wineboot for wine installations.

### PR DESCRIPTION
**Pull request by Etaash-mathamsetty**
This lets me use my custom wine builds again! (after symlinking wineserver)

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by flavioislima — COMMENTED** [review]
I think this will fail if wineboot does not exists:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/9212526c7a6f3be470d26751c0d567dffc1765bc/src/backend/launcher.ts#L475

**Comment by Etaash-mathamsetty** [comment]
> I think this will fail if wineboot does not exists:
> 
> https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/9212526c7a6f3be470d26751c0d567dffc1765bc/src/backend/launcher.ts#L475

if the wine installation is valid running wine wineboot --init will work fine

File: src/backend/config.ts [hunk 1]
```diff
@@ -393,24 +393,17 @@ abstract class GlobalConfig {
   }
 
   /**
-   * Checks if a Wine version has Wineboot/Wineserver executables and returns the path to those if they're present
+   * Checks if a Wine version has the Wineserver executable and returns the path to those if they're present
```
Comment (CommandMC) [inline src/backend/config.ts:396]:
```suggestion
   * Checks if a Wine version has the Wineserver executable and returns the path to it if it's present
```

File: src/backend/config.ts [hunk 2]
```diff
@@ -393,24 +393,17 @@ abstract class GlobalConfig {
...
    * @param wineBin The unquoted path to the Wine binary ('wine')
-   * @returns The quoted paths to wineboot and wineserver, if present
+   * @returns The quoted paths to wineserver, if present
```
Comment (CommandMC) [inline src/backend/config.ts:398]:
```suggestion
   * @returns The quoted path to wineserver, if present
```

File: src/backend/config.ts [hunk 3]
```diff
@@ -393,24 +393,17 @@ abstract class GlobalConfig {
...
    */
-  public getWineExecs(wineBin: string): {
-    wineboot: string
-    wineserver: string
-  } {
+  public getWineExecs(wineBin: string): { wineserver: string } {
```
Comment (CommandMC) [inline src/backend/config.ts:400]: Somewhat unnecessary now to return an object with just one key
Might as well just return the path directly

Comment (Etaash-mathamsetty) [reply src/backend/config.ts:400]: this required more code changes, and I just found it simpler to leave it the way it is, but I can it change if needed

**Review by flavioislima — APPROVED** [review]
LGTM

**Review by flavioislima — APPROVED** [review]
LGTM

**Pull request merged**